views/partials/footer.ejs

```diff
@@ -0,0 +1,55 @@
+<!--footer bar begins-->
+<footer class="footerbar" role="footer">
+    <div class="footerbar-container">
+        <div class="footer-container">
```

Comment:
@ZM202 This tag has a class that is blank so it's not really do anything. This can be removed along with the matching closing tag.

Reply:
All set.